frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/events-card.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { EventsBody } from '@console/internal/components/dashboard/events-card/events-body';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import {
+  EventModel,
+  PersistentVolumeClaimModel,
+  PersistentVolumeModel,
+} from '@console/internal/models';
+import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
+import { EventKind } from '@console/internal/module/k8s';
+import { cephStorageNamespace } from '../../../constants/index';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+
+const ocsEventNamespaceFilter = (event: EventKind): boolean =>
+  _.get(event, 'metadata.namespace') === cephStorageNamespace ||
```

Comment:
you could reuse `getNamespace` from `@console/shared`

Reply:
why not to put the body of function directly into `ocsEventFilter` ? Or split the function to `ocsEventNamespaceFilter` and `ocsEventKindFilter`